_source/logzio_collections/_metrics-sources/Prometheus-cloudwatch-exporter.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Ship Prometheus metrics from AWS CloudWatch with the OpenTelemetry Collector
+logo:
+  logofile: prometheusio-icon.svg
+  orientation: vertical
+data-source: Prometheus
+templates: ["docker-metricbeat"]
+open-source:
+  - title: 
+    github-repo: 
+contributors:
+  - yotamloe
+  - yberlinger
+shipping-tags:
+  - prometheus
+---
+
+# logzio-aws-metrics
+
+**IMPORTANT ⚠️ :** This project is intended for specific private beta customers.
+
+This project allows you to collect Prometheus format metrics from Amazon CloudWatch with the CloudWatch exporter, and ship them to Logz.io using the OpenTelemetry collector.
+
+### Schema
+![Prometheus-CloudFront-OpenTelemetry schema](https://dytvr9ot2sszz.cloudfront.net/logz-docs/metrics-prometheus/aws-docker-v2.png)
+
+### **Note:** 
+* Extra calls and charges on AWS API requests may be generated by this project. 
+
+* You can track the number of AWS API requests with the `cloudwatch_requests_total` metric, which provides labels with the API name and namespace specification. For example : 
+`cloudwatch_requests_total{action="getMetricStatistics",namespace="AWS/EC2",} 876.0`).
+
+* For more information about the prom/cloudwatch-exporter cost, refer to the relevant [Prometheus documentation](https://github.com/prometheus/cloudwatch_exporter#cost) and to the [AWS cloudwatch API pricing page](https://aws.amazon.com/cloudwatch/pricing/).
+
+#### Before you begin
+
+<div class="tasklist">
+
+##### Set up your IAM user
+
+You'll need an [IAM user](https://console.aws.amazon.com/iam/home)
+with these permissions:
+
+* `cloudwatch:GetMetricData`
```

Comment:
These are the minimum permissions:
cloudwatch:ListMetrics
cloudwatch:GetMetricStatistics
tag:GetResources

Reply:
Done in next commit